packages/pwa/app/commerce-api/auth.js

```diff
@@ -79,23 +78,54 @@ class Auth {
     }
 
     get authToken() {
-        return this._authToken
+        return this._storage.get(tokenStorageKey)
+    }
+
+    set authToken(token) {
+        this._storage.set(tokenStorageKey, token)
     }
 
     get refreshToken() {
-        return this._refreshToken
+        return (
+            this._storage.get(refreshTokenStorageKey) ||
+            this._storage.get(refreshTokenGuestStorageKey)
+        )
+    }
+
+    set refreshToken(obj = {}) {
```

Comment:
The setter accepts an object, but the getter returns a string. That's weird, and means that this does not work:
```ts
this.refreshToken = this.refreshToken
```

Reply:
👍